src/event_form/steps/event-form-step3.html

```diff
@@ -26,6 +26,9 @@ <h2 class="title-border">
           <div class="alert alert-danger" role="alert" ng-show="cityAutoCompleteError">
             Er was een probleem tijdens het ophalen van de steden
           </div>
+            <p>&nbsp;</p>
+            <p>&nbsp;</p>
+            <p>&nbsp;</p>
```

Comment:
@stijnswaanen ik zou de min-height van het '<div id="gemeente-kiezer" ng-hide="selectedCity !== ''">' element aanpassen in plaats van '<p>&nbsp;</p>' toe te voegen. Die 'min-height' zou je ook in de media-queries kunnen aanpassen naar gelang de behoeften van het scherm.